### ISSUE TITLE
Using CR around the metadata ignores the metadata

### ISSUE THREAD

**Issue by r0n0c** [notable/notable#868]
- **OS Version**: Win10 x64 & Mac OS 10.15
- **Notable Version**:1.7.3

### Current behavior

I edited a Notable note with a Mac text editor that uses CR for the line breaks. When I then opened the same file in Notable (on Windows 10) it was moved to the 'Untagged' section with the metadata as part of the note and new metadata added to the top file with LF as the line break. 

When I edited the file in an editor to change the line breaks from CR to CR LF (to match the rest of the Windows file) and removed the newly added metadata Notable recognized the information and added it to the correct tags.

### Expected behavior

Notable to interpret CR, CR LF, and LF all the same

**Comment by fabiospampinato** [comment]
@r0n0c What text editor was it? I've encountered `\n` and `\r\n` newlines, but never `\r` alone.

According to Wikipedia `\r` or `\n\r` may also be used, but they only mention pretty obsolete systems using it. Regardless we should probably be more compatible in this regard.

**Comment by r0n0c** [comment]
I am putting this in here as my own note `\r` = CR  and `\n` = LF

I was using BBEdit on Mac when it broke the file. When I fixed the file it was on Windows 10 and I was using Notepad++ to change all the line brakes to CR LF. 

I just checked again all on Win10 this time with Notepad++ if you change the line breaks to `\r` The metadata is ignored, and the note moves to "Untagged". When you switch them back to `\n` or `\n \r` it reads the metadata again.
